### ISSUE TITLE
Write function to load data from Kasmi dataset

### ISSUE THREAD

**Issue by ADPriceless** [ADPriceless/TDA_FinalProject_Solar#9]
- To be done after #8
- This dataset contains more data than the other
- However, it is likely going to be more complicated to create a function to load it
- Therefore:
  - Use the Hou dataset for initial model training
  - Create this function later to try to improve the models (if they're under performing);
  - Or since Kasmi definitely has images without solar panels (whereas Hou may not), include this sooner?
  - (Put this function in training set only? Hou is categorised so there could more easily take a strafefied sample for test set. But if there's a wide enough distribution in Kasmi, it may not matter)
